WireDomain/Tests/WireDomainTests/Repositories/Mock/MockFeatureConfigRepositoryProtocol.swift

```diff
@@ -0,0 +1,136 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Combine
+import WireAPI
+import WireDataModel
+
+@testable import WireDomain
+
+class MockFeatureConfigRepositoryProtocol: FeatureConfigRepositoryProtocol {
```

Comment:
just wondering why this mock is not within AutoMockable.generated.swift

Reply:
the stencil doesn't handle generics, I had to create one manually

Reply:
@jullianm can you add a comment here in the file, it helps then to reassess if we can move to autogeneration or not

Reply:
fixed in https://github.com/wireapp/wire-ios/pull/1962/commits/29dd2ec245a18c6e60dc3ec8578f199b24dc24d4